recipes/freecad/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "freecad" %}
+{% set version = "0.18b0" %}
+{% set sha256 = "1462df34caa67791bb7512f6c464f16a15d49f0d9f777996cb4e7b8174c57191" %}
+{% set commit = "e49a5af3ae2f976017f1156bdc113594f81d3fb5" %}
+
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}_{{ version }}.tar.gz
+    url: https://github.com/FreeCAD/FreeCAD/archive/{{ commit }}.tar.gz
+    sha256: {{ sha256 }}
+    patches:
+        - osx.patch
+        - smesh.patch
+        - win_unicode.patch
+
+build:
+    number: 0
+    skip: True  # [py < 36]
+
+requirements:
+    build:
+        - {{ compiler("cxx") }}
+        - vc 14  # [win]
+        - ninja
+        - cmake
+    host:
+        - nomkl  # [unix]
+        - boost-cpp 1.66.0
```

Comment:
Remove the version here

Reply:
I have to overwrite the pinnings here. The dependencies are not yet updated to latest pinnings. (Same is true for hdf5 via vtk)

Reply:
What do you mean? boost-cpp is pinned at 1.66.0 in conda-forge-pinning

Reply:
ah nice, I thought it's already at 1.76... thanks for the information.